ui/app/src/components/DevContentOpsDashlet/DevContentOpsDashlet.tsx

```diff
@@ -44,52 +46,106 @@ export function DevContentOpsDashlet(props: DevContentOpsDashletProps) {
     stats: null,
     loading: false
   });
+  const [days, setDays] = useState(30);
   const onRefresh = useMemo(
-    () => () => {
-      setState({ stats: null, loading: true });
-      fetchPublishingStats(site, 30).subscribe((stats) => {
-        setState({ stats, loading: false });
+    () => (backgroundRefresh?: boolean) => {
+      if (!backgroundRefresh) {
+        setState({ stats: null, loading: true });
+      }
+      fetchPublishingStats(site, days).subscribe((stats) => {
+        setState({ stats, ...(!backgroundRefresh && { loading: false }) });
```

Comment:
That would set loading to true when `backgroundRefresh` is undefined> I could make it always set it to false but I was trying to avoid setting it to false (since it was already false)